ds3_java_cli/src/main/java/com/spectralogic/ds3cli/util/Utils.java

```diff
@@ -327,4 +327,73 @@ public static String getBuildDate(final Properties properties) {
         return properties.get("build.date").toString();
     }
 
+    /**
+     * parse a string to get absolute date
+     * @param diff format Yyear.Mmonth.Ddate.hhours.mminutes.sseconds e.g., "Y2016.M11.D10.h12"
+     * @return long time in ms
+     */
+    public static long parseParamDate(final String diff) throws java.text.ParseException {
```

Comment:
Rather than taking the input and converting the format, if you're already going through the trouble of parsing it, why not just capture the data you need and create the date object manually?